package.json

```diff
@@ -2,131 +2,127 @@
+    "copy-config": "cp -r api-server/config server/config",
+    "format:all": "prettier --config ./.prettierrc.yml --write './src/**/*'",
+    "i18n": "i18next \"src/**/*.{js,jsx,ts,tsx}\" [-oc] -c i18next-parser.config.js",
     "img": "echo quay.io/$QUAY_USER/$npm_package_name:$npm_package_version",
     "img-build": "podman build --pull -f Dockerfile --platform linux/amd64 -t quay.io/$QUAY_USER/$npm_package_name:$npm_package_version .",
     "img-push": "podman push quay.io/$QUAY_USER/$npm_package_name:$npm_package_version",
     "img-run": "podman run -it -p 9001:9001 quay.io/$QUAY_USER/$npm_package_name:$npm_package_version",
+    "lint": "yarn eslint src --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix",
+    "prepare": "husky install",
     "start": "node ./server/app.js ./dist",
-    "start-dev": "ts-node --project tsconfig-server.json ./api-server/app.ts ./dist",
     "start-console": "./start-console.sh",
     "start-console-tls": "./start-console-tls.sh",
-    "i18n": "i18next \"src/**/*.{js,jsx,ts,tsx}\" [-oc] -c i18next-parser.config.js",
-    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'",
-    "lint": "eslint ./src --fix && stylelint \"src/**/*.css\" --allow-empty-input --fix",
-    "chromatic": "chromatic",
+    "start-dev": "ts-node --project tsconfig-server.json ./api-server/app.ts ./dist",
     "test": "NODE_TLS_REJECT_UNAUTHORIZED=0 TZ=UTC jest --runInBand",
     "test:coverage": "yarn run test --watch=false --coverage",
     "test:generate-output": "yarn test -- --json --outputFile=.jest-test-results.json",
-    "prepare": "husky install",
-    "format:all": "prettier --config ./.prettierrc.yml --write './src/**/*'"
+    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}'"
   },
-  "dependencies": {
-    "@peculiar/x509": "^1.11.0",
-    "@tanstack/react-query": "^4",
-    "@types/base-64": "^1.0.2",
-    "@types/cors": "^2.8.17",
-    "@types/express": "^4.17.21",
-    "@types/jsonwebtoken": "^9.0.6",
-    "@types/node": "^20.10.4",
-    "@types/validator": "^13.11.7",
-    "base-64": "^1.0.0",
-    "connect": "^3.7.0",
-    "cors": "^2.8.5",
-    "dotenv": "^16.4.5",
-    "eslint-plugin-unused-imports": "^3.2.0",
-    "express": "4.18.2",
-    "express-openapi-validator": "5.1.2",
-    "express-rate-limit": "^7.2.0",
-    "history": "^4.9.0",
-    "i18next": "^23.8.2",
-    "jsonwebtoken": "^9.0.2",
-    "lodash-es": "^4.17.21",
-    "node-fetch": "2",
-    "openapi-to-md": "^1.0.24",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-router": "5.3.4",
-    "react-router-dom": "5.3.4",
-    "require-from-string": "^2.0.2",
-    "showdown": "^2.1.0",
-    "swagger-routes-express": "^3.3.2",
-    "validator": "^13.11.0",
-    "yaml": "^2.4.5"
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
+      "yarn run pretty-quick --staged",
+      "eslint --fix"
+    ]
   },
   "devDependencies": {
```

Comment:
Hello @lavocatt , is there any reason for moving all dependencies into the devDependencies section? Initially, I separated them to distinguish between dev and prod dependencies after discussing with Howard. so just curious to know.

Reply:
Because i referred some other projects and modified like that way. in our code also by separating both dev-dependcies and prod dependencies.

Reply:
This is from a discussion I had with @vojtech who is one of the maintainer from the dynamic plugin sdk. He told me that we shouldn't separate dependencies and devDependencies and instead to have everything in devDependencies. He said that the only reason why we would need dependencies is if the plugin would get imported by another one and published on npm.

Btw, I'm not sure neither if that is the correct way of doing things, it's just what the ui-extensibility team might want to see in our project when we request assistance from them.

Reply:
Oh okay, Thanks for the clarification Thomas!